tests/data/uhp.json

```diff
@@ -0,0 +1 @@
+null
```

Comment:
Hi @loopodoopo 
The file was not generated correctly. Please re-do it.
Thx

Reply:
This works fine: php ./scripts/collect-snmp-data.php -h 172.19.121.17 -v ""

but php ./scripts/collect-snmp-data.php -h 46 -v "" does not produce anything?